server/apps/newsletter/services.py

```
@@ -0,0 +1,74 @@
+    def delete_subscriber(self, email: str) -> bool:
+        """
+        Delete a subscriber from Beehiiv
+        """
+        endpoint = f"{self.base_url}/publications/{self.publication_id}/subscriptions/email:{email}"
+        
+        try:
+            response = requests.delete(endpoint, headers=self.headers)
+            response.raise_for_status()
+            return True
+        except requests.exceptions.RequestException as e:
+            raise Exception(f"Failed to delete Beehiiv subscriber: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)